pkg/imgpkg/image/dir_image_test.go

```diff
@@ -61,4 +66,39 @@ func TestDirImage(t *testing.T) {
 			})
 		}
 	}
+
+	t.Run("When extracting tar that contains files and folders with permissions", func(t *testing.T) {
+		if runtime.GOOS == "windows" {
+			t.Skip("permissions values are different in windows because of umask")
+		}
+		img, err := image.NewFileImage(filepath.Join("test_assets", "img_tar_with_permissions.tar"), nil)
+		require.NoError(t, err)
+		folder, err := os.MkdirTemp("", "test")
```

Comment:
non-blocked suggestion.

We might wanna use `T.TempDir()` instead of os.MkdirTemp and defer removal. It actually does it for you.